model_tests.rb

```diff
@@ -495,6 +495,15 @@ def test_heatexchanger_airtoair_sensibleandlatent_rb
     result = sim_test('heatexchanger_airtoair_sensibleandlatent.rb')
   end
 
+  def test_heatexchanger_desiccant_balancedflow_rb
+    result = sim_test('heatexchanger_desiccant_balancedflow.rb')
+  end
```

Comment:
Another question: I think there's also a relation to CoilSystemCoolingDXHeatExchangerAssisted and CoilSystemCoolingWaterHeatExchangerAssisted. We should test for these too no?

Reply:
Added two new tests for these.

Reply:
You also need to add it to autosize_hvac test since the PerfData can be autosized

Reply:
Meaning we don't want to autosize in tests outside of the autosize_hvac test?

Reply:
not necesarilly, meaning the autosize test will pick up that it was not added but can be autosized and will fail. Run it locally and see for yourself

The autosize test is responsible for making sure the SQL query to retrieve the autosized value are working correcltly.

The autosize test: https://github.com/NREL/OpenStudio-resources/blob/9eea2df0f5b8dfc07892e6b159d43d8bed8c6dda/model_tests.rb#L1351-L1354

 `/path/to/openstudio model_tests.rb -n test_autosizing_rb`

Reply:
This makes me realize I should modify the pull_request.yml workflow to always run that one...

Reply:
Done in b84ffa54324fce288e6023219965e8fbbcb8bbb3. I merged develop in, CI should report on it.